### PR TITLE
feat(web-app): library details, add resources section

### LINKED ISSUE #578: Add resource links to library detail page
After talking with @aubrey-oneal - I confirmed that we don't need the body copy (D) in the design reference below for now.

**Inheritance**
- [x] Instead of showing "Depends on", let's keep it simple with "Inherits" if the library uses the `inherits` schema key
- [x] Only show the version if specified in the inherits key (see https://github.com/carbon-design-system/carbon-platform/blob/main/docs/resource-schemas.md#library-inheritance)
- [x] The library icon is available here and needs added it to the platform icons package https://ibm.ent.box.com/file/887827201496

**Source**
- [x] Instead of "Source", use "GitHub" or "IBM GitHub" to dual-purpose and the label and the host

**Package**
- [x] Only show if the package.json does not have `private: true` (see comment below)
- [x] The package icon is available here and needs added to the platform icons package https://ibm.ent.box.com/file/887827201496

**Design ref**
image

**Internal code packages**

We'll be indexing IBM Internal libraries that don't publish packages to npm. Instead, they often publish to artifactory or some other internal registry.

Looking at CloudPal for example, the package.json doesn't specify anywhere that it publishes to artifactory. If this package.json had `private: true` (as it should), then we know to not try to link out to npm. https://github.ibm.com/ibmcloud/pal/blob/master/patterns/package.json

### PR DESCRIPTION
**Pull request by francinelucca**
Closes #578 

- [x] Fix Margin Top Styling
- [x] Confirm npmjs button

### Added

- black npm logo

### Removed

- red npm logo

### Testing

http://localhost:3000/assets/carbon-charts-angular

### CODE REVIEW

**Review by alisonjoseph — APPROVED** [review]

File: services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js [hunk 1]
```diff
@@ -145,6 +168,35 @@ const Library = ({ libraryData, params, navData }) => {
             </Column>
           </Dashboard>
         </Column>
+        <Column sm={4} md={8} lg={8}>
+          <section>
+            <h2 className={styles['resources-heading']}>Resources</h2>
```
Comment (alisonjoseph) [inline services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js:173]: This works fine as is, but wondering if we want to use the <H2> component here (everywhere) to get the autolink heading functionality?

**Review by alisonjoseph — CHANGES_REQUESTED** [review]
Realized the h2 isn't bringing in the correct type token, have a couple suggestions to add.

File: services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js (hunk 1 above)
Comment (alisonjoseph) [inline services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js:173]:
```suggestion
            <H2>Resources</H2>
```

File: services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js [hunk 2]
```diff
@@ -6,19 +6,22 @@
...
-import { Svg64Community } from '@carbon-platform/icons'
+import { Svg32Github, Svg32Library, Svg64Community } from '@carbon-platform/icons'
 import clsx from 'clsx'
 import { get } from 'lodash'
 import { useRouter } from 'next/router'
 import { NextSeo } from 'next-seo'
 import { useContext, useEffect } from 'react'
 import { libraryPropTypes, paramsPropTypes, secondaryNavDataPropTypes } from 'types'
 
+import CardGroup from '@/components/card-group'
```
Comment (alisonjoseph) [inline services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js:17]:
```suggestion
import { H2 } from '@/components/markdown'
import CardGroup from '@/components/card-group'
```

File: services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss [hunk 3]
```diff
@@ -32,3 +31,8 @@
     width: 50%;
   }
 }
+
+.resources-heading {
+  margin-top: spacing.$spacing-11;
+  margin-bottom: spacing.$spacing-07;
+}
```
Comment (alisonjoseph) [inline services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss:38]:
```suggestion
```

The H2 component should have this built in already.

Comment (francinelucca) [reply services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss:38]: thanks 🙏

Comment (alisonjoseph) [inline services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss:38]:
```suggestion
```
I don't think we need these styles

**Review by alisonjoseph — APPROVED** [review]
LGTM once extra styles are removed 😄

File: services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss (hunk 3 above)
Comment (francinelucca) [reply services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.module.scss]: woops! my bad I thought I commited them the first time 🤦‍♀️

**Review by andreancardona — APPROVED** [review]
💯 looks good!

**Pull request merged**